packages/devtools_app/lib/src/error_badge_manager.dart

```diff
@@ -23,6 +23,7 @@ class ErrorBadgeManager extends DisposableController
     NetworkScreen.id: ValueNotifier<int>(0),
     LoggingScreen.id: ValueNotifier<int>(0),
   };
+  final _activeErroredWidgets = ValueNotifier<List<InspectableWidgetError>>([]);
```

Comment:
rename to `_activeErrors` and make it a `Map<String, ValueNotifier<List<DevToolsError>>>` where `DevToolsError` is some base class that takes a message and a unique id. `InspectableWidgetError` can be an extension of this class. 

This will allow the functionality you are building to be extended to other screens for which we'll want to track unique errors.

Reply:
Done in 6f779c11, although because we can't cast `ValueNotifier <List<DevToolsError>>` to `ValueNotifier <List<InspectableWidgetError>>` we have to call `.cast()` on the list when rendering the errors. I'm not sure if there's a way around that.